app/assets/stylesheets/snowcrash/base.scss

```diff
@@ -312,6 +312,14 @@ body {
 
     .node-header {
 //      height: $breadCrumbHeight;
+
+      // Styling of the breadcrumbs menu items conflicts when trying to match
+      // dropdown style of the main navigation. The dropdown ends up always
+      // pulling right, off the screen. This pulls it left, and makes it visible
+      .dropdown-menu-more {
+        left: auto;
+        right: 0;
+      }
```

Comment:
I don't think making a new class for this is necessary. Simply adding a `pull-right` class to the `<ul>` element will achieve this.